src/test/java/software/amazon/smithy/lsp/RequestBuilders.java

```diff
@@ -250,4 +256,24 @@ public DidChangeWatchedFilesParams build() {
             return new DidChangeWatchedFilesParams(changes);
         }
     }
+
+    public static final class DidChangeWorkspaceFolders {
+        final List<WorkspaceFolder> added = new ArrayList<>();
```

Comment:
Maybe I just overlooked, but Im not seeing where these are used as package private?

Reply:
it looks to me like these builder members can just be private.

Reply:
Made them all private